raster-foundry/raster-foundry
# Add index on datasource column in scenes table

## Overview

This PR adds an index on datasource column on scenes table

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~


## Testing Instructions

 * Run migrations and see if an index is added to the datasource column in scenes table
 * Tested with @notthatbreezy on staging -- adding this index can fix the time out issue querying for scenes before deleting datasources.

Closes https://github.com/azavea/raster-foundry-platform/issues/426
